test/jdk/javax/net/ssl/DTLS/CipherSuite.java

```diff
@@ -65,6 +68,8 @@ public class CipherSuite extends DTLSOverDatagram {
     private static boolean reenable;
 
     public static void main(String[] args) throws Exception {
+        SecurityUtils.removeFromDisabledTlsAlgs("TLS_RSA_*");
```

Comment:
Instead, you can add the "re-enable" argument to the @run lines starting with "TLS_RSA_".

Reply:
Indeed, done!